src/Modules/OrchardCore.Commerce/Views/BooleanProductAttributeField.cshtml

```diff
@@ -8,9 +9,9 @@
 }
 
 <input type="hidden" name="line.Attributes[@index].Key" value="@attr.PartName.@attr.Name" />
-<input type="hidden" name="line.Attributes[@index].Value" value="false" />
```

Comment:
Seems like a duplicate of the input directly below. It was causing invalid HTML (`error: Duplicate form control name "line.Attributes[1].Value" (form-dup-name)`) and reoving it doesn't seem to cause additional issues based on my testing.

Reply:
This is the equivalent of how `@Html.CheckBoxFor()` works by default so the form value is sent even for false. But as pointed out [here](https://github.com/OrchardCMS/OrchardCore.Commerce/issues/365#:~:text=The%20attribute%20should%20be%20applied%20even%20if%20its%20value%20is%20false.) it doesn't work so it's fine to remove.